internal/network/command_dns_record_list.go

```diff
@@ -26,10 +26,10 @@ func (c *command) newDnsRecordListCommand() *cobra.Command {
 		),
 	}
 
-	cmd.Flags().String("gateway", "", "Gateway ID.")
+	cmd.Flags().StringSlice("gateways", nil, "A comma-separated list of gateway IDs.")
 	cmd.Flags().StringSlice("names", nil, "A comma-separated list of display names.")
 	cmd.Flags().StringSlice("resource-ids", nil, "A comma-separated list of resource IDs.")
```

Comment:
```suggestion
	cmd.Flags().StringSlice("resources", nil, "A comma-separated list of resource IDs.")
```

Reply:
We also have `--gateways` in this command, not `--gateway-ids`. Also, we'll want to support resource names in the future, I think

Reply:
Good catch, thanks. This also would have conflicted with the upcoming V4 change of removing unnecessary `-id` suffixes.